cmd/importer/rand.go

```
@@ -80,7 +81,7 @@ func randDate(col *column) string {
 
 	minTime, err := time.Parse(dateFormat, min)
 	if err != nil {
-		log.Warnf("randDate err %s", err)
+		log.Warn("randDate", zap.Error(err))
```

Do we need these warning msgs? @lamxTyler

---

Cureently yes. Maybe it is better to return the error to caller, but it is another issue.

---

any better idea for the error message?

---

"parse min date failed"?

---

Done.
PTAL @lamxTyler @XuHuaiyu